src/world/Util/ActorFilter.h

```diff
@@ -29,18 +30,15 @@ namespace Sapphire::World::Util
     bool conditionApplies( const Entity::Actor& actor ) override;
   };
 
-  // usage in psudocode
-  //
-  //  std::set< ActorPtr > filterActorList( inputSet, filter )
-  //  {
-  //    std::set< ActorPtr > resultSet;
-  //    for( every actor in inputSet )
-  //    {
-  //       if( filter.conditionApplies( actor ) )
-  //         resultSet.insert( actor.asPointer() );
-  //    }
-  //    return resultSet;
-  //
+  /////////////////////////////////////////////////////////////////////////////
+
+  class ActorFilterSingleTarget : public ActorFilter
+  {
+    uint32_t m_actorId;
+  public:
+    explicit ActorFilterSingleTarget( uint32_t actorId );
+    bool conditionApplies( const Entity::Actor& actor ) override;
+  };
```

Comment:
I don't think we want to filter when we already know the affected target. This would be a waste of time.